### PR TITLE
[hotfix] Clean up redundant code in Manifests Table.

### PR DESCRIPTION
**Pull request by zhuangchong**
### Purpose

Clean up redundant code in Manifests Table.

### CODE REVIEW

File: paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java [hunk 1]
```diff
@@ -233,6 +232,10 @@ private StatsManifestsGetter(FileIO fileIO, Path location, Table dataTable) {
 
         private void initialize() {
```
Comment (JingsongLi) [inline paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java:233]: Just convert `StatsManifestsGetter` to a static method?

Comment (JingsongLi) [reply paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java:233]: I mean we can do a bit of refactoring, here the class doesn't seem to be of much use, just a static method will do for our needs.

Comment (zhuangchong) [reply paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java]: Sorry, I misunderstood, I have added a static method.

File: paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java [hunk 2]
```diff
@@ -219,34 +218,20 @@ private InternalRow toRow(ManifestFileMeta manifestFileMeta) {
     }
 
     private static class StatsManifestsGetter {
```
Comment (JingsongLi) [inline paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java:220]: You can just remove this class.

Comment (zhuangchong) [reply paimon-core/src/main/java/org/apache/paimon/table/system/ManifestsTable.java]: OK, Thanks.

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**